apps/e2e/ag-grid-storybook/src/app/shared/font-loading/font-loading.service.ts

```diff
@@ -1,10 +1,12 @@
 import { Injectable } from '@angular/core';
-import design from '@blackbaud/skyux-design-tokens/json/design-tokens.json';
+import * as _design from '@blackbaud/skyux-design-tokens/json/design-tokens.json';
 
-import FontFaceObserver from 'fontfaceobserver';
+import * as FontFaceObserver from 'fontfaceobserver';
 import { Observable, from } from 'rxjs';
 import { map } from 'rxjs/operators';
 
+const design = _design;
+
```

Comment:
Is there a reason to do this?

Reply:
Unfortunately, yes. I've added a comment to explain.